### PR TITLE
fix(@angular-devkit/build-angular): script chunk in angular.json return false then calling isInitial

### LINKED ISSUE #13223: Webpack extra script chunk return canBeInitial as false.
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
```

### Command (mark with an `x`)
```
- [x] build
```

### Versions

node --version
v10.12.0
npm --version
6.4.1
Angular CLI: 7.1.1
Node: 10.12.0
OS: darwin x64
Angular: 7.1.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.11.1
@angular-devkit/build-angular     0.11.1
@angular-devkit/build-optimizer   0.11.1
@angular-devkit/build-webpack     0.11.1
@angular-devkit/core              7.1.1
@angular-devkit/schematics        7.1.1
@angular/elements                 7.1.3
@ngtools/webpack                  7.1.1
@schematics/angular               7.1.1
@schematics/update                0.11.1
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.23.1

### Repro steps
When using ngx-build-plus with html-webpack-plugin
The Global scripts chunk is not isInitial.
Since the extra chunk in ScriptsWebpackPlugin has not add with entrypoint,
therefore the html-webpack-plugin will nerver add the script to the html (3.2.0).

### Desired functionality
The lazy options should pass to ScriptsWebpackPlugin.
It the script is not lazy than 
`chunk.addGroup(entrypoint);`

ps: html-webpack-plugin have remove this feather in master but it should leave the option to plugin developer.

### Mention any other details that might be useful
Root cause
https://github.com/jantimon/html-webpack-plugin/blob/v3.2.0/index.js#L111
https://github.com/jantimon/html-webpack-plugin/blob/v3.2.0/index.js#L384

### PR DESCRIPTION
**Pull request by Tsuki**
Fixes #13223

### CODE REVIEW

**Comment by mgechev** [comment]
@Tsuki thanks for the PR!

File: packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts [hunk 1]
```diff
@@ -83,7 +84,9 @@ export class ScriptsWebpackPlugin {
 
     const entrypoint = new EntryPoint(this.options.name);
     entrypoint.pushChunk(chunk);
-
+    if (!this.options.lazy){
+      chunk.addGroup(entrypoint);
```
Comment (clydin) [inline packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts]: Lazy shouldn’t be a guard for this.   The chunk is already part of the group from the pushChunk call.  The issue here is that it was assumed that the pushChunk call performs the reciprocal action.  The correct behavior is to always execute both actions.

Deciding which entrypoints should be present and how they are represented in an HTML fileis a function of the html processing plugin.

Comment (Tsuki) [reply packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts]: In current situation, the chunk [canBeInitial](https://github.com/webpack/webpack/blob/master/lib/Chunk.js#L199) always return false when not put the ChunkGoup in to the chunk. 
The chunk will [filter out](https://github.com/jantimon/html-webpack-plugin/blob/v3.2.0/index.js#L384) even you stat in the filter option.

Comment (clydin) [reply packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts]: Yes.  The only change that is need here is the addition of `chunk.addGroup(entrypoint);`.  The lazy logic is only relevant to the HTML plugin.

Comment (Tsuki) [reply packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts]: ok changed

**Pull request merged**